tests/load/pipeline/test_scd2.py

```diff
@@ -0,0 +1,370 @@
+# timezone is removed from all datetime objects in these tests to simplify comparison
+
+import pytest
+from typing import List, Dict, Any
+from datetime import datetime, timezone  # noqa: I251
+
+import dlt
+from dlt.common.pipeline import LoadInfo
+from dlt.common.schema.typing import DEFAULT_VALIDITY_COLUMN_NAMES
+from dlt.common.normalizers.json.relational import DataItemNormalizer
+from dlt.common.normalizers.naming.snake_case import NamingConvention as SnakeCaseNamingConvention
+from dlt.destinations.sql_jobs import HIGH_TS
+
+from tests.pipeline.utils import assert_load_info
+from tests.load.pipeline.utils import (
+    destinations_configs,
+    DestinationTestConfiguration,
+    load_tables_to_dicts,
+)
+
+
+ACTIVE_TS = datetime.fromisoformat(HIGH_TS.isoformat()).replace(tzinfo=None)
```

Comment:
Can't we just use `HIGHT_TS.replace(tzinfo=None)`?

Reply:
`HIGH_TS` is a `pendulum` `DateTime` because the `datetime` module is a banned import. However, when fetching data from the destination tables to assert its content, timestamps are returned as `datetime` objects. Hence the conversion from `DateTime` to `datetime.`